spec/tree/research/researches/Research.js

```diff
@@ -0,0 +1,41 @@
+import Research from "../../../../src/tree/research/researches/Research";
```

Comment:
Nothing for this line of code.
However, the filename of this file should be `ResearchSpec.js`, right?

Reply:
You are right! 🙂